.drone.yml

```diff
@@ -492,6 +492,39 @@ steps:
     HOST: grafana-server
   image: cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97
   name: end-to-end-tests-various-suite
+- commands:
+  - apt-get update && apt-get install git curl jq --yes
```

Comment:
maybe you can skip the `update`?

Reply:
Most likely yep, it's a force of habit to run update before install. If this proves to be stable then my plan for the future would be to bundle the image ourselves to make the E2E tests more efficient.